cirq/testing/equals_tester.py

```diff
@@ -33,8 +33,8 @@ class EqualsTester:
     def __init__(self):
         self.groups = [(_ClassUnknownToSubjects(),)]
 
-    def add_equality_group(self, *group_items: Any):
-        """Tries to add a disjoint equivalence group to the equality tester.
+    def verify_equality_group(self, *group_items: Any):
```

Comment:
Don't bother having this be a separate method; just stick with `add_equlity_group` being the only one and use that.

Or is this intended to be an internal helper? In that case prefix it with `_`.

Reply:
I wanted to run base class method (verify equality) prior to running the method from the child class (verify ascending).

Reply:
Ah, I see. Yes, it does make more sense in the context of a child class trying to avoid creating equivalence groups behind the scenes. But OrderTester shouldn't inherit EqualsTester so that becomes irrelevant.